garage/envs/box2d/box2d_env.py

```diff
@@ -124,13 +123,13 @@ def observation_space(self):
 
     @property
     def action_bounds(self):
-        return bounds(self.action_space)
+        return self.action_space.low, self.action_space.high
 
     def forward_dynamics(self, action):
-        if len(action) != flat_dim(self.action_space):
+        if len(action) != np.prod(self.action_space.low.shape):
```

Comment:
Does it break if we impose the stronger (and simpler) constraint: `action.shape == self.action_space.low.shape`?. There's not much point in having action spaces if they are not followed.